src/decaf-qt/gamelistworker.cpp

```diff
@@ -222,6 +172,7 @@ ForeachDirectoryEntry(uint64_t* numEntriesOut,
    // windows loop
```

Comment:
Should be using Qt to do this.

http://doc.qt.io/qt-5/qdiriterator.html
http://doc.qt.io/qt-5/examples-xml.html
http://doc.qt.io/qt-5/qxmlsimplereader.html